README.md

```diff
@@ -78,13 +80,13 @@ The url is formed by the host and port defined in the previous step.
 **Note:** In order to keep compatibility with previous versions, v2.0 keep attending the default connector ("Connector") in the endpoint 0.0.0.0:8089. This will be removed in future versions.   
 
 ```
-# The prom2teams endpoint to send HTTP POST requests to.
+// The prom2teams endpoint to send HTTP POST requests to.
 url: 0.0.0.0:8089/v2/<Connector1>
 ```
 
 ### Templating
 
-prom2teams provides a [default template](app/teams/template.j2) built with [Jinja2](http://jinja.pocoo.org/docs/2.9/) to render messages in Microsoft Teams. This template could be overrided using the 'templatepath' argument ('--templatepath <Jinja2 template file path>') during the application start.
+prom2teams provides a [default template](resources/templates/teams.j2) built with [Jinja2](http://jinja.pocoo.org/docs/2.9/) to render messages in Microsoft Teams. This template could be overrided using the 'templatepath' argument ('--templatepath <Jinja2 template file path>') during the application start.
```

Comment:
Jinja2 link should point to 2.10 version: http://jinja.pocoo.org/docs/2.10/ (as is defined in _requirements.txt_ file)

Reply:
ok